### ISSUE TITLE
Missing documentation for :isset, :length, :each, and @random.

### ISSUE THREAD

**Issue by Lex-2008** [pocketbase/site#23]
You've made a great progress since I was here last time! (around version 0.5, I think)

However, I noticed that some of features introduced in [v0.11.0][v] seem to be not reflected in the documentation, namely `:isset`, `:length`, and `:each` modifiers, and probably the `@random` sort key. Or, rather, they are better documented in changelog rather than in docs. :)

[v]: https://github.com/pocketbase/pocketbase/releases/tag/v0.11.0

Would you like me to try to make a draft PR adding some kind of explanation for them?

Once again, thank you for your continuous work on this project and making it more and more awesome every day!

**Comment by ganigeorgiev** [comment]
Yes, unfortunately those are currently not documented. I was postponing it because I was planning to do it together with some other tasks, but I'll think in the next couple days how to structure it and at least add some notes about the modifiers in the "API rules and filters" docs.

---

_(Side-note with a little more info)_

Initially, I was planning a docs refactoring with the [JS app hooks task](https://github.com/orgs/pocketbase/projects/2?pane=issue&itemId=17629907) for v0.13/v0.14, but I've postponed it because before introducing the hooks we need to do another small refactoring of the internals and also eventually abstract the echo router to minimize the API surface and the implementation details that will leak in the JS wrappers.

I want to take my time with it and things may slow down a little starting from next month since I'm planning finally to start working on my other oss project (Presentator), so that I can have a real/practical experience with PocketBase with a mid-sized project in order to understand better the possible use cases and eventual pitfalls (_so far I've built only one small internal application with PocketBase at my last job couple months ago_).
